src/utils/airtable.js

```diff
@@ -47,25 +47,17 @@ export default function getComponentDetails(componentName) {
 // this function should return title and image data for all components
 export async function getHomePageInfo() {
   // connect to our table 'Components
-  const records = await base('Components').select().firstPage();
+  const records = await base('Components')
+    .select({
+      sort: [{ field: 'Component Name', direction: 'asc' }],
```

Comment:
Thanks for thinking of sorting even though I forgot to mention that detail in the card!

Reply:
Weirdly enough it was adding the Accordion as the last item even though it's the first one in the list so I threw it in there!